docs/guides/other_libs/serilog.md

```diff
@@ -0,0 +1,100 @@
+---
+uid: Guides.OtherLibs.Serilog
+title: Serilog
+---
+
+# Configuring serilog
+
+## Prerequisites
+
+- A basic working bot with a logging method as described in [Creating your first bot](xref:Guides.GettingStarted.FirstBot)
+
+## Installing the Serilog package
+
+You can install the following packages through your IDE or go to the nuget link to grab the dotnet cli command.
+
+|Name|Link|
+|--|--|
+|`Serilog.Extensions.Logging`| [link](https://www.nuget.org/packages/Serilog.Extensions.Logging)|
+|`Serilog.Sinks.Console`| [link](https://www.nuget.org/packages/Serilog.Sinks.Console)|
+
+## Configuring Serilog
+
+Serilog will be configured at the top of your async Main method, it looks like this
+
+```cs
+using Discord;
+using Serilog;
+using Serilog.Events;
+
+public class Program
+{
+  public static Task Main(string[] args) => new Program().MainAsync().GetAwaiter().GetResult();
+
+  public async Task MainAsync()
+  {
+    Log.Logger = new LoggerConfiguration()
+        .MinimumLevel.Verbose()
+        .Enrich.FromLogContext()
+        .WriteTo.Console()
+        .CreateLogger();
+
+    _client = new DiscordSocketClient();
+
+    _client.Log += LogAsync;
+
+    //  You can assign your bot token to a string, and pass that in to connect.
+    //  This is, however, insecure, particularly if you plan to have your code hosted in a public repository.
+    var token = "token";
+
+    // Some alternative options would be to keep your token in an Environment Variable or a standalone file.
+    // var token = Environment.GetEnvironmentVariable("NameOfYourEnvironmentVariable");
+    // var token = File.ReadAllText("token.txt")[0];
```

Comment:
My own mistake on initial review. ReadAllText returns string, ReadAllLines returns array. [0] is not required.